dashboard/README.md

```diff
@@ -1,5 +1,5 @@
 ## Next.js App Router Course - Build a Dashboard
 
-This repository contains the starter templates for the [Next.js App Router Course](), separated by chapters.
+This repository contains the starter templates for the [Next.js App Router Course](xxx), separated by chapters.
 
-For more information, see the [course curriculum]() on the Next.js Website.
+For more information, see the [course curriculum](xxx) on the Next.js Website.
```

Comment:
Should both of these links just be `https://nextjs.org/learn`?

Reply:
```suggestion
For more information, see the [course curriculum](https://nextjs.org/learn) on the Next.js Website.
```